doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -21,6 +21,75 @@ This specification is licensed under the
 [FIWARE Open Specification License]
 (http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_Open_Specification_Legal_Notice_%28essential_patents_license%29).
 
+## FIWARE NGSI information and data model introduction
+
+The FIWARE NGSI (Next Generation Service Interface) API defines 
+
+* a **data model** for context information, based on a simple information model using the notion of *context entities*
+* a **context data interface** for exchanging information by means of query, subscription, and update operations
+* a **context availability interface** for exchanging information on how to obtain context information
+(whether to separate the two interfaces is currently under discussion).
+
+### Context Entities
+
+Context entities, or simply called entities, are the center of gravity
+in the FIWARE NGSI information model. An entity represents a
+real-world object like e.g. a sensor, a person, or a room. Each entity
```

Comment:
Actually, an entity can represent things of logical nature that strictly speking are not part of the real-works, e.g. an "alarm" entity within a trouble ticketing system.

However, this is a very minor comment, and we can adjust the description in a next PR.

NTC